src/PTFAnalysis.cpp

```diff
@@ -336,30 +358,147 @@ void PTFAnalysis::FitWaveform( int wavenum, int nwaves, PTF::PMT pmt) {
 
     double fit_minx = min_bin - 40.0;
     double fit_maxx = min_bin + 8.0*2.5;
-    
 
-    if( ffitfunc == nullptr ) ffitfunc = new TF1("mygauss",funcEMG,fit_minx-30,fit_maxx+30,5);
-    ffitfunc->SetParameters( fitresult->amp, fitresult->mean, 8.0, 1.0, fitresult->ped );
-    ffitfunc->SetParNames( "Amplitude", "Mean", "Sigma", "exp decay", "Offset" );
+    int fitstat;
+    double amplitude;
 
-    
-    ffitfunc->SetParameter(0, -2);
-    ffitfunc->SetParameter(1, min_bin );
-    ffitfunc->FixParameter(2, 10.9 );
-    ffitfunc->FixParameter(3, 0.5 );
-    //    ffitfunc->SetParameter(4, 0.99 );
-    if(pmt.channel == 0)
-      ffitfunc->FixParameter(4, 0.999);
-    else if(pmt.channel == 1)
-      ffitfunc->FixParameter(4, 1.0036);
-    ffitfunc->SetParLimits(0, -100, 0);
-    ffitfunc->SetParLimits(1, 2022.0, 2358.0 );
-    //ffitfunc->SetParLimits(2, 10.56, 10.58 );
-    //ffitfunc->SetParLimits(3, 0.1, 0.9 );
-    //    ffitfunc->SetParLimits(4, 0.99, 1.01 );
+    // ellipitcall modified gaussian
+    if(pmt.channel >= 16){
```

Comment:
Hmm, I can clean this up, but all this code is very tentative right now.  We don't know the right functional form and the functional form  is different for different channels on the same electronics card.  The code for analyzing this data will remain pretty messy and in a state of flux.

For the moment I really can't easily clean this up.  I could remove it from the pull request but I'd just need to recommit it afterwards, because the different channels of the cable really are different.  The functional form of the second revision card will be different again.